Library/Homebrew/utils/json.rb

```diff
@@ -1,4 +1,4 @@
-require "vendor/okjson"
+require "json"
 
 module Utils
   module JSON
```

Comment:
`stringify_keys` doesn't seem to be JSON-specific so could be moved (and we could vendor some of ActiveSupport for such things if we really wanted) and will need to check the exceptions behave correctly but otherwise should be pretty simple.